flake.nix

```diff
@@ -0,0 +1,100 @@
+{
+  description = "categorifier";
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-21.11";
+    concat = {
+      url = "github:con-kitty/concat/wavewave-flake";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+  };
+  outputs = { self, nixpkgs, concat }:
+    let
+      pkgs = import nixpkgs {
+        overlays = [ concat.overlay ];
+        system = "x86_64-linux";
```

Comment:
We shouldn't hardcode the system. E.g., I do a lot of dev on macos with nix-darwin.

Reply:
Right. I will update the expression with flake-utils.eachSystem. Thanks for the catch.
https://github.com/numtide/flake-utils#usage